GenderPayGap.WebUI/Services/OrganisationService.cs

```diff
@@ -134,6 +125,49 @@ public Organisation ImportOrganisationFromCompaniesHouse(string companyNumber, U
             return organisation;
         }
 
+        private Organisation SetDeletedRetiredOrganisationToActive(Organisation existingOrganisation, User requestingUser)

```

Comment:
Let's say an organisation had been deleted/retired before the latest scopes were added. Does the organisation have all the scopes?

I assume no scopes are added unless an organisation is active. If so, we need to add any missing scopes here as well.

Reply:
Again, looks like this is handled by one of the recurring jobs SetPresumedScopesJob (04:50 each day) that also doesn't have any filters for retired/deleted organisation to add new OrganisationScopes, so they should be there as presumed in/out.

Reply:
Interesting!